src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -739,7 +739,7 @@ private async Task<ConnectBlocksResult> ReconnectOldChainAsync(List<ChainedHeade
 
             // We failed to jump back on the previous chain after a failed reorg.
             // And we failed to reconnect the old chain, database might be corrupted.
-            this.logger.LogError("A critical error has prevented reconnecting blocks");
+            this.logger.LogError("A critical error has prevented reconnecting blocks, error = {0}", connectBlockResult.Error);
```

Comment:
Is LogError also logged with debug log level? To me, these critical errors should be in the normal node log.